src/ledger-db/initdb/1_create_transactions.sh

```diff
@@ -36,14 +36,15 @@ fi
 readonly ENV_VARS=(
   "POSTGRES_DB"
   "POSTGRES_USER"
+  "POSTGRES_PASSWORD"
   "LOCAL_ROUTING_NUM"
 )
 
 
 add_transaction() {
     DATE=$(date -u +"%Y-%m-%d %H:%M:%S.%3N%z" --date="@$(($6))")
     echo "adding demo transaction: $1 -> $2"
-    psql -X -v ON_ERROR_STOP=1 -v fromacct="$1" -v toacct="$2" -v fromroute="$3" -v toroute="$4" -v amount="$5" --username "$POSTGRES_USER" --dbname "$POSTGRES_DB" <<-EOSQL
+    PGPASSWORD="$POSTGRES_PASSWORD" psql -X -v ON_ERROR_STOP=1 -v fromacct="$1" -v toacct="$2" -v fromroute="$3" -v toroute="$4" -v amount="$5" --username "$POSTGRES_USER" --dbname "$POSTGRES_DB" <<-EOSQL
```

Comment:
What happens if postgres is deployed without a helm chart for example outside the scope of this work? Will that variable interfere with it, or will it get ignored?

Reply:
This change is not related to helm chart. The problem here is that if you run this script on a Postgres with a password this script will not work because `psql` will fail with an `unauthorized` error. 
This var will not get ignored. I think it should work if `POSTGRES_PASSWORD` is defined but unfortunately, I don't know how to verify this.